sections/05_treat.md

```diff
@@ -209,9 +218,9 @@ actives and inactives to other inactives is less important
 reviewing more existing literature on the topic` `TODO: check which other
 existing reviews should be referenced`
 
-We primarily focus on ligand-based approaches that train on chemicals' features
-without requiring knowledge of the target, as opposed to alternative strategies
-that use target features such as the protein structure `TODO: add examples`.
+Here we primarily focus on ligand-based approaches that train on chemicals' features
+without requiring prior knowledge of the target, as opposed to strategies
+specifically targeting features such as protein structure `TODO: add examples`.
```

Comment:
Maybe "specifically modeling"?  I'm not sure whether "target" (as in protein target) and "targeting" could be confusing.